build.gradle

```diff
@@ -667,6 +667,7 @@ project(':iceberg-gcp') {
     api project(':iceberg-api')
     implementation project(':iceberg-common')
     implementation project(':iceberg-core')
+    implementation libs.caffeine
```

Comment:
We probably need to double check License/Notice since this is a new addition to the bundle.

Reply:
I've removed this dependency